grand_challenge_forge/utils.py

```diff
@@ -33,3 +35,28 @@ def directly_import_module(name, path):
     spec.loader.exec_module(module)
 
     return module
+
+
+def slugify(title):
```

Comment:
I already commented on the GC PR, but can you explain why you need this instead of just passing along the actual algorithm slug with the context?

Reply:
James is also pushing to use the slug, so I'll switch to those.